samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml

```diff
@@ -0,0 +1,216 @@
+#
```

Comment:
Same comment as above for all OCP samples

Reply:
@rajendraindukuri : So all the OCP samples will get added later not now? If yes, we need to make sure to update the latest SHA of SDC in the storage_csm_powerflex_v2110.yaml file whenever it will get added.

Reply:
@suryagupta4 All OCP samples will be updated after publishing to Redhat. If we need to update 1.11 sample, we can directly update here https://github.com/dell/csm-operator/tree/main/samples/ocp/1.6.0 (which is on main branch) if the patch release branch is not carved out of main.

Reply:
yes, the patch release branch is not carved out of main, the SDC latest SHA needs to be updated in main branch as well. We will raise another PR for main to update that. @rishabhatdell: Please create a PR for updating the SDC SHA against main branch.

For this patch release, we created `ocp/1.6.1` folder inside which all the samples are present which are required for patch release (which are powerstore and unity v2.11.1) and powerflex v2.11.0 was also added since it will have the updated SHA id for SDC. If the samples have to be removed, shall we create the `ocp/1.6.1` and keep the samples in the main branch?

Reply:
Removed the ocp/1.6.1 samples as the YAMLs will be updated after the image is published to Red Hat by @rajendraindukuri